src/libraries/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -1381,6 +1381,34 @@ private object DebuggerDisplayContent
             object IDebuggerDisplay.Content { get { return DebuggerDisplayContent; } }
         }
         #endregion
+
+        #region ReceiveAllAsync
+
+#if NETSTANDARD2_1
```

Comment:
This should be separated out into its own partial file.

Reply:
Moved to a dedicated partial file!